src/main/scala/com/monsanto/arch/cloudformation/model/resource/IAM.scala

```diff
@@ -70,8 +70,8 @@ case class `AWS::IAM::ManagedPolicy`(
   Description:    Option[String] = None,
   Path:           Option[String] = None,
   Groups:         Option[Seq[ResourceRef[`AWS::IAM::Group`]]] = None,
-  Roles:          Option[Seq[ResourceRef[`AWS::IAM::Group`]]] = None,
-  Users:          Option[Seq[ResourceRef[`AWS::IAM::Group`]]] = None,
+  Roles:          Option[Seq[ResourceRef[`AWS::IAM::Role`]]] = None,
+  Users:          Option[Seq[ResourceRef[`AWS::IAM::User`]]] = None,
```

Comment:
I'm not sure why these were both `Group` before. This will break backwards compatibility but I'm guessing it wouldn't have really worked anyways.